utils/converter.go

```diff
@@ -31,6 +32,23 @@ func ConvertMapToString(mapString map[string]string) (string, error) {
 	return string(stringPractitionerLinks), nil
 }
 
+// ConvertStringToArray is a helper function to split string to array string
+func ConvertStringToArray(stringItem string, splitChar string) []string {
```

Comment:
This function does not seem to be used (merged in with the next one?) - if so, should be deleted along with the test for it.

Reply:
Fixed